docs/course_materials/exercises/test_exercises/03_Variables_and_datatypes/test_homework_variables.py

```diff
@@ -0,0 +1,73 @@
+"""
+Automatically check student's homework
+----------------------------------------
+The tests will check:
+ 1. The content of the exercise file
+ 2. The execution and output of the exercise file
+ 3. The output with different inputs
+
+Guidelines:
+- You have to specify the dir Student's exercise files should be placed
+in the same dir as this test.
+- Test names are in the form: test_<exercise>.py
+- Before each test should be the appropriate fixture: @pytest.mark.<topic>
+"""
+import pytest
+
+
+@pytest.mark.variables
+def test_variables_1(helpers):
+    helpers.student_file = helpers.home_dir + 'variables_1.py'
+    helpers.expected_pycode = [
+        [r'^.*x *= *9.*', 'x variable is not defined'],
+        [r'^.*y *= *7.*', 'y variable is not defined'],
+        [r'^.*z *= *x *\+ *y.*', 'z variable is not defined'],
+        [r'^.*print\(.*z.*\).*', 'print message is missing'],
+    ]
+    helpers.expected_stdout = [[r'.*16', 'The expected output is 16']]
+
+    helpers.test_assignment()
+
+
+@pytest.mark.variables
+def test_variables_2(helpers):
+    helpers.student_file = helpers.home_dir + 'variables_2.py'
+    helpers.expected_pycode = [
+        [r'^.*\*.*', '* sign is missing'],
+        [r'^.*-.*', '- sign is missing'],
+        [r'^.*/.*', '/ sign is missing'],
+        [r'^.*%.*', '%' + 'sign is missing'],
```

Comment:
It would great if you can add a space before, because the it prints 
"%sign is missing"
Also, if you could change the error to "The Arithmetic Operator % is missing" it would be great (for all the lines above also)